<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Library</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="LibraryPage" name="libraryPage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>421</width>
      <height>501</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="page_main">
     <widget class="myList" name="bookList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>421</width>
        <height>451</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="myBook_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>My Book</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_bookinfo">
     <widget class="BookInfo" name="bookInfo">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>411</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="lastPage_button">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Last Page</string>
      </property>
     </widget>
     <widget class="QPushButton" name="borrow_button">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>470</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Borrow</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_edit">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>470</y>
        <width>81</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>EDIT</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_mybook">
     <widget class="MyBook" name="myBook">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>411</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="LastPage1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>470</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Last Page</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_history">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>470</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>History</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_advanced">
     <widget class="QPushButton" name="pushButton_addbook">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>50</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>ADD BOOK</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_HISTORY">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>HISTORY</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_insertbook">
     <widget class="QPushButton" name="pushButton_addConfirm">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>450</y>
        <width>111</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Confirm</string>
      </property>
     </widget>
     <widget class="QGroupBox" name="groupBox_editbook">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>401</width>
        <height>371</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="label_12">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>330</y>
         <width>61</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Status</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_2">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>320</y>
         <width>291</width>
         <height>31</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Available</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Borrowed</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Lost</string>
        </property>
       </item>
      </widget>
      <widget class="QLineEdit" name="lineEdit_title">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>20</y>
         <width>291</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_6">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>90</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Author</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>30</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_8">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>210</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Pages</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_7">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>150</y>
         <width>71</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Genre</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_9">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>270</y>
         <width>81</width>
         <height>21</height>
        </rect>
       </property>
       <property name="text">
        <string>Publisher</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_author">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>80</y>
         <width>291</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_genre">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>140</y>
         <width>291</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_pages">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>200</y>
         <width>291</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_publisher">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>260</y>
         <width>291</width>
         <height>31</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="pushButton_delete">
      <property name="geometry">
       <rect>
        <x>170</x>
        <y>450</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
     <zorder>groupBox_editbook</zorder>
     <zorder>pushButton_addConfirm</zorder>
     <zorder>pushButton_delete</zorder>
    </widget>
    <widget class="QWidget" name="page_history_search">
     <widget class="MyBook" name="myBook_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>70</y>
        <width>411</width>
        <height>431</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_usernameSearch">
      <property name="geometry">
       <rect>
        <x>80</x>
        <y>0</y>
        <width>251</width>
        <height>31</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>0</y>
        <width>101</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Username</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_enter">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>0</y>
        <width>61</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>Enter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>40</y>
        <width>181</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Name</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="page_edit"/>
   </widget>
   <widget class="QGroupBox" name="groupBox_indentity">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>70</y>
      <width>90</width>
      <height>84</height>
     </rect>
    </property>
    <property name="title">
     <string>Identity</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QRadioButton" name="radioButton_staff">
       <property name="text">
        <string>Staff</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radioButton_reader">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="text">
        <string>Reader</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="LogInBox">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>50</y>
      <width>243</width>
      <height>221</height>
     </rect>
    </property>
    <property name="title">
     <string>LogIn</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_username">
         <property name="text">
          <string>Username</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_username"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_password">
         <property name="text">
          <string>Password </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEdit_password"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="logIn_button">
       <property name="text">
        <string>log in</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="SignInButton">
       <property name="text">
        <string>Sign up</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="login_label">
       <property name="text">
        <string>...</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="SignUpBox">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>290</y>
      <width>301</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>SignUp</string>
    </property>
    <widget class="QPushButton" name="SignUpConfirmButton">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>210</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Confirm</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_namesignup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_usernamesignup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_passwordsignup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>150</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_reentersignup">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>151</width>
       <height>22</height>
      </rect>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>91</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Your name</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>58</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Username</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>58</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>Password</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Re-enter password</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_username_check">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>100</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Check</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_check">
     <property name="geometry">
      <rect>
       <x>167</x>
       <y>104</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="SearchEngine" name="lineEdit_search">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>10</y>
      <width>521</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="SearchButton">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>10</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Search</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>10</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>title</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>author</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>genre</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>publisher</string>
     </property>
    </item>
   </widget>
   <widget class="QPushButton" name="pushButton_advanced">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>250</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Advanced</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_main">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Main</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myList</class>
   <extends>QListWidget</extends>
   <header>mylist.h</header>
  </customwidget>
  <customwidget>
   <class>LibraryPage</class>
   <extends>QStackedWidget</extends>
   <header>librarypage.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BookInfo</class>
   <extends>QTextEdit</extends>
   <header>bookinfo.h</header>
  </customwidget>
  <customwidget>
   <class>MyBook</class>
   <extends>QTableWidget</extends>
   <header>mybook.h</header>
  </customwidget>
  <customwidget>
   <class>SearchEngine</class>
   <extends>QLineEdit</extends>
   <header>searchengine.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
